nestkernel/simulation_manager.cpp

```diff
@@ -40,16 +40,16 @@
 #include "psignal.h"
 
 nest::SimulationManager::SimulationManager()
-  : simulating_( false )
-  , clock_( Time::tic( 0L ) )
+  : clock_( Time::tic( 0L ) )
   , slice_( 0L )
   , to_do_( 0L )
   , to_do_total_( 0L )
   , from_step_( 0L )
   , to_step_( 0L ) // consistent with to_do_ == 0
   , t_real_( 0L )
-  , terminate_( false )
+  , simulating_( false )
```

Comment:
We should try to simplify this multivariable state machine, and we should document what the state graph is.